src/sentry/static/sentry/app/components/events/interfaces/frame/frame.jsx

```diff
@@ -381,24 +384,36 @@ const RepeatedContent = styled(VertCenterWrapper)`
   justify-content: center;
 `;
 
-const NativeLineContent = styled(VertCenterWrapper)`
-  flex: 1;
-  overflow: hidden;
-  justify-content: center;
-
-  & > span {
-    display: block;
-    padding: 0 5px;
+const PackageInfo = styled('div')`
+  display: flex;
+  order: 2;
+  @media (min-width: ${props => props.theme.breakpoints[2]}) {
+    order: 0;
   }
+  display: grid;
+  grid-template-columns: auto 1fr;
```

Comment:
🤔 is it flex or display?

Reply:
do you mean grid? display:flex was there by mistake... I just removed it 😉